package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "mcp-storage-server",
+  "version": "1.0.0",
+  "description": "Storacha MCP Storage Server - Decentralized storage for AI systems",
+  "type": "module",
+  "main": "index.ts",
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index.js",
+    "start:stdio": "MCP_TRANSPORT_MODE=stdio node dist/index.js",
+    "start:sse": "MCP_TRANSPORT_MODE=sse node dist/index.js",
+    "dev": "tsc-watch --onSuccess \"node dist/index.js\"",
+    "clean": "rm -rf dist",
+    "lint": "eslint src --ext .ts",
+    "test": "vitest run",
+    "test:watch": "vitest watch",
+    "test:coverage": "vitest run --coverage",
+    "inspect": "pnpm build && npx @modelcontextprotocol/inspector node dist/index.js",
+    "inspect:sse": "pnpm build && MCP_TRANSPORT_MODE=sse npx @modelcontextprotocol/inspector node dist/index.js",
+    "inspect:stdio": "pnpm build && MCP_TRANSPORT_MODE=stdio npx @modelcontextprotocol/inspector node dist/index.js"
+  },
+  "keywords": [
+    "mcp",
+    "storage",
+    "storacha",
+    "decentralized",
+    "distributed",
+    "storage-server",
+    "long-term-storage",
+    "ai"
+  ],
+  "author": "Storacha <dev@storacha.network>",
+  "license": "MIT & Apache-2.0",
+  "files": [
+    "dist"
+  ],
+  "dependencies": {
+    "@ipld/car": "^5.4.0",
+    "@modelcontextprotocol/sdk": "^1.7.0",
+    "@ucanto/core": "^10.3.1",
+    "@ucanto/interface": "^10.2.0",
+    "@ucanto/principal": "^9.0.2",
+    "@web3-storage/w3up-client": "^17.2.0",
```

Comment:
Should be safe to use `@storacha/client` now.

Reply:
I will add that in another round: https://github.com/storacha/mcp-storage-server/issues/7